lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -234,6 +234,8 @@ function dosomething_user_form_alter(&$form, $form_state, $form_id) {
     break;
 
     case 'user_profile_form':
+      $form['#validate'][] = 'dosomething_user_validate_password_length';
+      dosomething_northstar_update_user($form_state);
```

Comment:
i don't think we want to break these out to different cases
also that `dosomething_northstar_update_user ` should be running as a submit hook already [here](https://github.com/chloealee/dosomething/blob/password-reset-flow/lib/modules/dosomething/dosomething_user/dosomething_user.module#L258)